spec/models/post_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) { create(:user) }
+
+  it 'is valid with valid attributes' do
+    post = Post.new(title: 'A Title', author: user, comments_counter: 0, likes_counter: 0)
+    expect(post).to be_valid
+  end
+
+  it 'is not valid without a title' do
+    post = Post.new(author: user, comments_counter: 0, likes_counter: 0)
+    expect(post).to_not be_valid
+  end
+
+  it 'is not valid if comments_counter is negative' do
+    post = Post.new(title: 'A Title', author: user, comments_counter: -1, likes_counter: 0)
+    expect(post).to_not be_valid
```

Comment:
- [ ] Some of your tests are not running as you can see on the screenshot below:
![Screenshot from 2023-10-27 14-05-38](https://github.com/arnoldnekemiah/blog/assets/63087491/176af36f-181d-49e3-ae7f-94350fd8d0d1)